conf/db/migration/metrics_portal_ddl/h2/V4__create_etag_in_alerts_and_expressions.sql

```diff
@@ -0,0 +1,15 @@
+-- Alerts etag --
+CREATE SEQUENCE portal.alerts_etag_seq START WITH 1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE;
+-- Trigger to be executed after each insert, update and delete statement --
+CREATE TRIGGER update_alerts_etag
+AFTER INSERT, DELETE, UPDATE ON portal.alerts
+FOR EACH ROW
+CALL "com.arpnetworking.database.h2.triggers.AlertsUpdateEtagTrigger";
```

Comment:
I see. I assume this is what prevents parameterization of the trigger with the sequence name? We could still refactor the code to have one logical instance and two stub subclasses.